src/python/test/test_cover_complex.py

```diff
@@ -8,8 +8,11 @@
       - YYYY/MM Author: Description of the modification
 """
 
-from gudhi import CoverComplex
+from gudhi import NGIComplex
+from gudhi.sklearn import CoverComplex
 import pytest
+import numpy as np
+from sklearn.cluster import AgglomerativeClustering
```

Comment:
As you introduce a new dependency on this test, in `src/python/CMakeLists.txt`, you should say:
```
      if(SKLEARN_FOUND)
        add_gudhi_py_test(test_cover_complex)
      endif()
```

Reply:
Thanks! Done.